packages/valory/skills/abstract_round_abci/tests/test_io/test_ipfs.py

```diff
@@ -33,10 +37,21 @@
 )
 
 
-ipfs_daemon = pytest.mark.usefixtures("ipfs_daemon")
+use_ipfs_daemon = pytest.mark.usefixtures("ipfs_daemon")
+
+
+@pytest.fixture(scope="module")
+def ipfs_daemon() -> Iterator[bool]:
+    """Starts an IPFS daemon for the tests."""
+    print("Starting IPFS daemon...")
+    daemon = IPFSDaemon()
+    daemon.start()
+    yield daemon.is_started()
+    print("Tearing down IPFS daemon...")
+    daemon.stop()
```

Comment:
Why not in the test tools?

Reply:
This belongs to ipfs plugin, which we'll do in the next release. Till then this has to be here